tchannel-core/src/main/java/com/uber/tchannel/api/errors/TChannelError.java

```diff
@@ -31,6 +31,9 @@
     public static final String ERROR_CONNECTION_FAILURE = "tchannel.socket";
     public static final String ERROR_NO_PEER_AVAILABLE = "tchannel.no-peer-available";
 
+    public static final String ERROR_CODEC = "tchannel.codec";
```

Comment:
the convention doesn't use "error" in the error string, e.g., "tchannel.connection.timeout"